localization/languages/bg.json

```diff
@@ -27,7 +27,7 @@
         "goForward": "Отиване една страница напред",
         "inspectElement": "Изследване на елемента",
         /* searchbar */
-        "pasteAndGo": null, //missing translation
+        "pasteAndGo": "Поставяне и посещаване", //missing translation
```

Comment:
Can you remove the "missing translation" comments?

Reply:
Oh yeah, I completely forgot to remove them, give me 5 min

Reply:
Done